Documentation/Features.md

```diff
@@ -0,0 +1,34 @@
+# WheelMUD Features
```

Comment:
How many of these features are fully tested and validated?  I suspect our actual feature list is much smaller based on my discovery of the inherited code so far.